lib/simulation/openfisca/mapping/index.js

```diff
@@ -100,6 +100,25 @@ var applyRessources = function(src, dest, schema, dateDeValeur) {
     });
 };
 
+var setNonInjectedPrestaToZero = function(dest, dateDeValeur){
+    var moisCourant = moment(dateDeValeur).format('YYYY-MM');
+
+    var prestationsInjectees = _.pick(PRESTATIONS, function(format) {
```

Comment:
If you iterate on “prestations”, the iteratee is probably “prestation” rather than “format”.